README.md

```diff
@@ -35,6 +35,13 @@ performing the action.
 
 You can always map multiple key bindings to the same snapshot name.
 
+- `@named-snapshot-switch-client`  
```

Comment:
nitpick: would `@named-snapshot-keytable` would be a better naming?

`switch-client` is a command in tmux to switch a client to a different session, with an option to change 'key table'. And what we does in this MR is to support a custom key table.